docs/src/UserGuide/BuildingLibrary.md

```diff
@@ -1,61 +1,91 @@
+Those using Mac OS X can install CMake through the use of a command-line tool
+called `brew`, as follows:
 
 ```sh
 brew install cmake
 ```
 
-It should not even be necessary to install anything else on top of macOS, since the compiler comes preinstalled. There is no need to read the next subsection ([Developer prerequisites](#developer-prerequisites)) if you are not intending to modify the core library and can safely jump to the section [Library Dependencies](#library-dependencies)
+It should not even be necessary to install anything else on top of macOS, since
+the compiler comes preinstalled. There is no need to read the next subsection
+([Developer prerequisites](#developer-prerequisites)) if you are not intending
+to modify the core library and can safely jump to the section
+[Library Dependencies](#library-dependencies)
 
 ### On Ubuntu 20.04
 
-Additionally to CMake, you will also require Clang in order to build Ubuntu 20.04. You can accomplish this by following the steps below:
+Additionally to CMake, you will also require Clang in order to build Ubuntu
+20.04. You can accomplish this by following the steps below:
 
 ```sh
 apt install clang-11 cmake
 ```
 
-You can skip straight to [Library Dependencies](#library-dependencies) if you do not intend to work on developing the core library.
+You can skip straight to [Library Dependencies](#library-dependencies) if you do
+not intend to work on developing the core library.
 
 ## Developer prerequisites
 
-Development of the core library requires additional tools. These are used formatting, linting and managing code quality. For this purpose, the following tools and packages are used:
+Development of the core library requires additional tools. These are used
+formatting, linting and managing code quality. For this purpose, the following
+tools and packages are used:
 
 - Python 3
 - Python packages specified in `requirements.txt`
 - clang-format
 - clang-tidy
 
-The installation process varies depending on the platform you use. The following subsections provide details on how to install these tools on each platform.
+The installation process varies depending on the platform you use. The following
+subsections provide details on how to install these tools on each platform.
 
 ### On Mac OS X
 
 TODO:
 
 ### On Ubuntu 20.04
 
-Installing the clang tools on Ubuntu along with Python can be accomplished by running these commands:
+Installing the clang tools on Ubuntu along with Python can be accomplished by
+running these commands:
 
 ```sh
 apt install clang-format-11 clang-tidy-11
 apt install python3 python3-pip
 ```
 
-We recommend that you use version 11 of clang in order to be consistent with the version of LLVM on which the library depends. In general, the code should work with any version of clang. Make sure that the compiler environment variables are defined correctly in order to select the correct version:
+We recommend that you use version 11 of clang in order to be consistent with the
```

Comment:
Similarly here, it may be good to use consistent casing for "Clang" versus "clang" when used as a name in prose.